cli/cmd/brokers.go

```diff
@@ -40,22 +85,25 @@ var describeCmd = &cobra.Command{
 	},
 }
 
-var listCmd = &cobra.Command{
+var listBrokersCmd = &cobra.Command{
 	Use:   "list",
 	Short: "List brokers the service controller knows about",
 	Long:  "List brokers the service controller knows about",
 	RunE: func(cmd *cobra.Command, args []string) error {
 		fmt.Printf("doing list %s %d\n", controller, timeout)
-		return callService("/v2/service_brokers", "GET", "list service brokers", nil)
+		return callService(BROKERS_URL, "GET", "list service brokers", nil)
 	},
 }
 
-var deleteCmd = &cobra.Command{
+var deleteBrokersCmd = &cobra.Command{
 	Use:   "delete",
 	Short: "Delete a broker from the service controller",
 	Long:  "Delete a broker from the service controller",
 	RunE: func(cmd *cobra.Command, args []string) error {
-		fmt.Printf("doing delete %s %d\n", controller, timeout)
-		return nil
+		if len(args) == 0 {
+			return fmt.Errorf("need uuid of the broker to delete")
```

Comment:
hmm, this is kind of interesting. The other cmds take a name but here we ask for a uuid. Perhaps we should, under the covers, convert the name to a uuid for them?